### PR TITLE
Fixes gondolapods, other centcom_podlauncher misc fixes

### LINKED ISSUE #43033: Player-controlled gondola pods cannot open themselves
@MrDoomBringer 
```
[00:03:04] Runtime in ,: bad arg name 'manual'
  proc name: open (/obj/structure/closet/supplypod/open)
  usr: spacemaniac/(gondola)
  usr.loc: (Arrival Shuttle (27, 62, 2))
  src: the gondola (/obj/structure/closet/supplypod/centcompod)
  src.loc: null
  call stack:
  the gondola (/obj/structure/closet/supplypod/centcompod): open(the gondola (/mob/living/simple_animal/pet/gondola/gondolapod), null, null)
  the gondola (/mob/living/simple_animal/pet/gondola/gondolapod): Release Contents()
```

```dm
	linked_pod.open(src, manual = TRUE)
```

```dm
/obj/structure/closet/supplypod/open(atom/movable/holder, var/broken = FALSE, var/forced = FALSE)
```

### PR DESCRIPTION
**Pull request by MrDoomBringer**
## About The Pull Request
Fixes Gondolapods, also lets people see if the gondolapod has made a delivery yet

## Why It's Good For The Game
Gondolapods are great
also tfw u were gonna just fix one issue but then u find 20 more while doing so

## Changelog
:cl: MrDoomBringer
fix: Gondolapods work again
fix: the see-through pod effect for centcom podlaunchers works again
fix: the Stealth effect for centcom podlaunchers works again
admin: sparks will not generate if the quietLanding effect is on, for the centcom podlauncher
admin: makes input text clearer for the centcom podlauncher

/:cl:

-----
(ed. SpaceManiac)
fixes #43033

### CODE REVIEW

**Comment by SpaceManiac** [comment]
does this fix #43033?

**Comment by DaxDupont** [comment]
Speed merge game breaking bug for Adminbus

File: code/modules/cargo/gondolapod.dm [hunk 1]
```diff
@@ -39,15 +39,22 @@
 	set name = "Release Contents"
 	set category = "Gondola"
 	set desc = "Release any contents stored within your vast belly."
-	linked_pod.open(src, manual = TRUE)
+	linked_pod.open(src, forced = TRUE)
+
+/mob/living/simple_animal/pet/gondola/gondolapod/examine(mob/user)
+	..()
+	if (contents.len)
+		to_chat(user, "<span class='notice'>It looks like it hasn't made its delivery yet.</b>.<span>")
```
Comment (ShizCalev) [inline code/modules/cargo/gondolapod.dm]: extra period

File: code/modules/cargo/gondolapod.dm [hunk 2]
```diff
@@ -39,15 +39,22 @@
...
 	set desc = "Release any contents stored within your vast belly."
-	linked_pod.open(src, manual = TRUE)
+	linked_pod.open(src, forced = TRUE)
+
+/mob/living/simple_animal/pet/gondola/gondolapod/examine(mob/user)
+	..()
+	if (contents.len)
+		to_chat(user, "<span class='notice'>It looks like it hasn't made its delivery yet.</b>.<span>")
+	else
+		to_chat(user, "<span class='notice'>It looks like it has already made its delivery.</b>.<span>")
```
Comment (ShizCalev) [inline code/modules/cargo/gondolapod.dm]: extra period

File: code/modules/cargo/supplypod.dm [hunk 3]
```diff
@@ -224,8 +227,8 @@
 
 /obj/effect/DPfall/Initialize(dropLocation, obj/structure/closet/supplypod/pod)
 	if (pod.style == STYLE_SEETHROUGH)
-		pixel_x = 0
-		pixel_y = 0
+		//pixel_x = 0
+		//pixel_y = 0
```
Comment (ShizCalev) [inline code/modules/cargo/supplypod.dm]: remove, don't comment out.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/modules/cargo/supplypod.dm (hunk 3 above)
Comment (MrDoomBringer) [reply code/modules/cargo/supplypod.dm]: oops did this during testing

**Comment by AnturK** [comment]
Fix the typos please.

**Comment by MrDoomBringer** [comment]
fuk

**Comment by MrDoomBringer** [comment]
@AnturK should be good

**Pull request merged**